doc/man/scons.xml

```diff
@@ -5810,90 +5762,89 @@ special variables for each command execution:</para>
   <listitem>
 <para>The file names of all sources of the build command
 that have changed since the target was last built.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>CHANGED_TARGETS</term>
   <listitem>
 <para>The file names of all targets that would be built
 from sources that have changed since the target was last built.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>SOURCE</term>
   <listitem>
 <para>The file name of the source of the build command,
 or the file name of the first source
 if multiple sources are being built.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>SOURCES</term>
   <listitem>
 <para>The file names of the sources of the build command.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>TARGET</term>
   <listitem>
 <para>The file name of the target being built,
 or the file name of the first target
 if multiple targets are being built.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>TARGETS</term>
   <listitem>
 <para>The file names of all targets being built.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>UNCHANGED_SOURCES</term>
   <listitem>
 <para>The file names of all sources of the build command
 that have
 <emphasis>not</emphasis>
 changed since the target was last built.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>UNCHANGED_TARGETS</term>
   <listitem>
 <para>The file names of all targets that would be built
 from sources that have
 <emphasis>not</emphasis>
 changed since the target was last built.</para>
-
   </listitem>
   </varlistentry>
 </variablelist>
 
-<para>Note that the above variables are reserved
```

Comment:
Not sure I'm a fan of removing the Note that's. They're meant to highlight an important fact..

Reply:
Hmmm.  I was intending to reword that, not drop it. Odd mistake to make.

Reply:
Okay, I didn't look before replying. Sure, can put back the "Note that".